### PR TITLE
[WIP] Added support for updating and deleting mentions

### PR DESCRIPTION
**Pull request by allouis**
🍦

### CODE REVIEW

File: ghost/core/core/server/services/mentions/BookshelfMentionRepository.js [hunk 1]
```diff
@@ -63,7 +63,9 @@ module.exports = class BookshelfMentionRepository {
      * @returns {Promise<Page<import('@tryghost/webmentions/lib/Mention')>>}
      */
     async getPage(options) {
-        const page = await this.#MentionModel.findPage(options);
+        // TODO: sqlite uses 0, mysql uses false... how do we construct this filter properly?
+        const filter = `deleted:[0,false]` + (options.filter ? `+(${options.filter})` : '');
```
Comment (9larsons) [inline ghost/core/core/server/services/mentions/BookshelfMentionRepository.js:67]: Not sure how we should handle `sqlite` using `0/1` vs. mysql using `true/false`.

File: ghost/webmentions/test/InMemoryMentionRepository.test.js [hunk 2]
```diff
@@ -76,4 +76,35 @@ describe('InMemoryMentionRepository', function () {
         assert(pageThree.meta.pagination.prev === 2);
         assert(pageThree.meta.pagination.next === null);
     });
+
+    describe(`GetPage`, function () {
```
Comment (9larsons) [inline ghost/webmentions/test/InMemoryMentionRepository.test.js:80]: This test doesn't really provide any code coverage but it confirm the behavior we want - that the front end will not receive deleted mentions.

File: ghost/core/core/server/services/mentions/BookshelfMentionRepository.js (hunk 1 above)
Comment (allouis) [reply ghost/core/core/server/services/mentions/BookshelfMentionRepository.js:67]: `true/false` _should_ work for SQLite! I believe knex will handle the difference

Comment (allouis) [reply ghost/core/core/server/services/mentions/BookshelfMentionRepository.js:67]: I've updated this to use the `enforcedFilters` method in models - which is used as the `overrides` property in `@tryghost/nql`

**Pull request merged**